osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/application/config/jms/JmsConfiguration.java

```diff
@@ -12,6 +12,14 @@
 import org.springframework.jms.core.JmsTemplate;
 import org.springframework.jms.listener.DefaultMessageListenerContainer;
 
+/**
+ * @deprecated Will be replaced by broker per queue configuration using the
+ *             {@link org.opensmartgridplatform.shared.application.config.messaging.JmsConfiguration}
+ *             interface and
+ *             {@link org.opensmartgridplatform.shared.application.config.messaging.JmsConfigurationFactory}
+ *             class
+ */
+@Deprecated
 public class JmsConfiguration {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1133)